test_setup.tendrl_httpd_ssl_selfsigned.yml

```diff
@@ -0,0 +1,17 @@
+---
+# ===============================================
+#  Setup of HTTPS for Tendrl UI, API and Grafana
+# ===============================================
+#
+# This is **minimal setup**, which **uses default local ssl certs** provided by
+# mod_ssl rpm package.
+
+- hosts: usm_server
+  remote_user: root
+  handlers:
+    - name: restart httpd
```

Comment:
The handler is already defined in the role, isn't it? Or did I miss something?

Reply:
good point